Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Screens/ScreenFactory/ScreenFactory.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Aquality.Appium.Mobile.Applications;
+using System;
+using System.Linq;
+using System.Reflection;
+
+namespace Aquality.Appium.Mobile.Screens.ScreenFactory
+{
+    /// <summary>
+    /// Abstract screen factory.
+    /// </summary>
+    /// <typeparam name="TPlatformScreen">Desired platform screen <see cref="IOSScreen"/> or <see cref="AndroidScreen"/></typeparam>
+    public abstract class ScreenFactory<TPlatformScreen> : IScreenFactory
+        where TPlatformScreen : class
+    {
+        /// <summary>
+        /// Returns an implementation of a particular app screen.
+        /// </summary>
+        /// <typeparam name="TAppScreen">Desired application screen.</typeparam>
+        public TAppScreen GetScreen<TAppScreen>()
+            where TAppScreen : IScreen
+        {
+            var type = Assembly.Load(AqualityServices.ApplicationProfile.AssemblyNameWithScreens)
```

Comment:
would it fail if AssemblyNameWithScreens was not specified? 
If it would, can we throw an understandable exception for this case?